fbw-common/src/systems/instruments/src/ND/shared/RwyAheadAdvisory.tsx

```diff
@@ -0,0 +1,58 @@
+import { DisplayComponent, EventBus, FSComponent, Subject, VNode } from '@microsoft/msfs-sdk';
```

Comment:
Please add copyright claim and SPDX license.

```
// Copyright (c) 2024 FlyByWire Simulations
// SPDX-License-Identifier: GPL-3.0
```

(this applies to multiple new files in the PR)